src/api/java/com/minecolonies/api/util/constant/SchematicTagConstants.java

```diff
@@ -14,4 +14,9 @@
     public static final String NORMAL_RAIDER = "normal";
     public static final String ARCHER_RAIDER = "archer";
     public static final String BOSS_RAIDER   = "boss";
+
+    /**
+     * Graveyard constants.
+     */
+    public static final String GRAVE       = "grave";
```

Comment:
```suggestion
    public static final String GRAVE        = "grave";
```
if you're trying to get it to be inline with the rest